Makefile.common

```diff
@@ -55,7 +55,7 @@ MKDIR:=install -d
 CORESRCDIRS:=\
   config 	lib		kernel		kernel/byterun	library \
   proofs 	tactics 	pretyping	interp		stm \
-  toplevel	parsing 	printing	intf	engine	ltac
+  toplevel	parsing 	printing	intf	engine	ltac vernac
```

Comment:
This is before all cosmetic, but for regularity and clarity of reading, I would preserve the aligned-column layout, and reorder the directories in the compilation order, ..., library, engine, pretyping, interp, proofs, ... as in CORECMA.